sw/device/tests/sim_dv/i2c_host_tx_rx_test.c

```diff
@@ -147,12 +147,12 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_i2c_host_set_enabled(&i2c, kDifToggleEnabled));
   CHECK_DIF_OK(
       dif_i2c_set_watermarks(&i2c, kDifI2cLevel30Byte, kDifI2cLevel4Byte));
+
+  for (uint32_t i = kDifI2cIrqFmtWatermark; i <= kDifI2cIrqTransComplete; ++i) {
+    CHECK_DIF_OK(dif_i2c_irq_set_enabled(&i2c, i, kDifToggleEnabled));
+  }
   CHECK_DIF_OK(
-      dif_i2c_irq_set_enabled(&i2c, kDifI2cIrqFmtWatermark, kDifToggleEnabled));
-  CHECK_DIF_OK(
-      dif_i2c_irq_set_enabled(&i2c, kDifI2cIrqRxWatermark, kDifToggleEnabled));
-  CHECK_DIF_OK(dif_i2c_irq_set_enabled(&i2c, kDifI2cIrqTransComplete,
-                                       kDifToggleEnabled));
+      dif_i2c_irq_set_enabled(&i2c, kDifI2cIrqSdaUnstable, kDifToggleDisabled));
```

Comment:
Looks like you could just reuse the array down here too.

Reply:
? aren't the indices different here?

Reply:
Ah, I guess I didn't pay close enough attention. Also... <insert complaint about having wide IP-specific interrupt vectors and CSRs in the PLIC _and_ the IP> ;)

Reply:
no worries, i can create two separate arrays as you say.

Reply:
updated